doc/UserGuide/InstallAndBasicUsage.rst

```diff
@@ -42,89 +48,27 @@ Alternatively if you want you can install it using pip to your local user enviro
-only download and install Ganga but also all the externals required as well. To run it, download it and do the
-following:
-
-.. code-block:: bash
-
-    wget http://ganga.web.cern.ch/ganga/download/ganga-install
-    python ganga-install --extern=GangaDirac <RELEASE>
-
-(Here <RELEASE> is in the format x.y.z)
-
-
-To update an install to the latest code form github develop (FOR DEVELOPERS ONLY, THIS IS UNSUPPORTED) try something similar to the following:
-
-.. code-block:: bash
-
-    wget http://ganga.web.cern.ch/ganga/download/ganga-install
-    chmod +x ganga-install
-    ./ganga-install 6.1.20
-    cd ~/Ganga/install
-    rm -fr 6.1.20
-    git clone https://github.com/ganga-devs/ganga
-    mv ganga 6.1.20
 
 From Github
 ^^^^^^^^^^^
 
-You can always download a ``.zip/.tar.gz`` of a release from github
-`here <https://github.com/ganga-devs/ganga/releases>`_ Simply unzip the file where you want Ganga to be available and
-then you can run it using:
-
-``<install-dir>/ganga-<RELEASE>/bin/ganga``
-
-Note that this will not install any of the additional packages Ganga requires so you may have import errors when
-running. If so, please install the appropriate python packages on your system.
+You can always install Ganga's most recent commit at the time of writing by running 
 
+``pip install -e git+git://github.com/ganga-devs/ganga.git#egg=ganga``
 
-For LHCb
-^^^^^^^^
-
-(This has been tested on lxplus, feel free to open an issue if this fails on cvmfs/CernVM)
-
-First grab the LHCb installer script:
-
-.. code-block:: bash
-
-    wget https://github.com/ganga-devs/lhcbsetupproject/raw/master/scripts/lhcb-prepare
-    chmod +x $TMPDIR/lhcb-prepare
-
-To install locally run one of the following:
-
-* For the official Ganga releases:
-.. code-block:: bash
-
-        ./lhcb-prepare v601r14
-
-* For an 'in development' release:
-.. code-block:: bash
-
-        ./lhcb-prepare -p v601r15
-
-* To keep the git history of the project so that you can develop patches and branches etc:
-.. code-block:: bash
-
-        ./lhcb-prepare -k -t v601r15
-
-You now have Ganga installed in ``$HOME/cmtuser/GANGA/GANGA_v601r15``
-
-* To update your install of Ganga to use the latest git code:
-.. code-block:: bash
-
-        cd $HOME/cmtuser/GANGA/GANGA_v601r15
-        git checkout develop
+If you don't have System Administrator rights or just want to do a private install then it can be beneficial to
+run using a ``virtualenv``:
 
-* Finally use:
 .. code-block:: bash
 
-        SetupProject ganga v601r15
+    # Create a virtualenv
+    virtualenv -ppython3 gangaenv
+    cd gangaenv/
+    . bin/activate
+    # Install Ganga 
+    pip install -e git+git://github.com/ganga-devs/ganga.git#egg=ganga
 
-to get a Ganga environment setup.
+Note that this will not install any of the additional packages Ganga requires so you may have import errors when running. If so, please install the appropriate python packages on your system.
```

Comment:
The command above should install the dependencies. Do you observe something else?

Reply:
The command was working just fine on my system, I wasn't so sure if it needed any additional packages. I should remove this part then, right?  

also I changed : pip install -e git+git@github.com:ganga-devs/ganga.git#egg=ganga 
to : pip install -e git+git://github.com/ganga-devs/ganga.git#egg=ganga
because it was throwing error. The latter worked fine.

Reply:
Yes, just remove the comment starting with "Note". And thanks for updating the command - the syntax I proposed is indeed old and has stopped working in recent pip versions.